imports/api/dashboards/server/methods.js

```diff
@@ -38,12 +68,123 @@ const addDashboard = new ValidatedMethod({
     if (meteorUser.profile.hoursToDays) {
       hoursToDays = meteorUser.profile.hoursToDays
     }
+    let hashedPassword = null;
+    if (password) {
+      hashedPassword = await bcrypt.hash(password, saltRounds);
+    }
     const _id = Random.id()
     await Dashboards.insertAsync({
-      _id, projectId, customer, startDate, endDate, resourceId, timeunit, hoursToDays,
```

Comment:
same as above I would appreciate it if we could keep customer and resourceId as optional parameters